lib/lndclient/LndClient.ts

```diff
@@ -551,21 +551,27 @@ class LndClient extends SwapClient {
     fee.setFixed(Math.floor(MAXFEE * request.getAmt()));
     request.setFeeLimit(fee);
 
+    let route: lndrpc.Route | undefined;
+
     try {
-      const routes = (await this.queryRoutes(request)).getRoutesList();
-      this.logger.debug(`got ${routes.length} route(s) to destination ${destination}: ${routes}, finalCltvDelta: ${finalCltvDelta}`);
-      return routes;
+      // QueryRoutes no longer returns more than one route
+      route = (await this.queryRoutes(request)).getRoutesList()[0];
```

Comment:
Why continue using a list if it has only one entry?

Reply:
This is because lnd still returns us a list: https://api.lightning.community/#grpc-response-queryroutesresponse